projects/cdk/utils/nodejs/normalize-common-js-import.ts

```diff
@@ -0,0 +1,6 @@
+export async function tuiNormalizeCommonJSImport<T>(
+    importPromise: Promise<T>,
+): Promise<T> {
+    // CommonJS's `module.exports` is wrapped as `default` in ESModule.
+    return importPromise.then((m: any) => (m.default || m) as T);
```

Comment:
Let's also use this new util here
https://github.com/Tinkoff/taiga-ui/blob/83a562e0bff72ad72f601a199e008d9696a7d287/projects/addon-doc/src/utils/raw-load.ts#L7-L9

Reply:
But we are return `*.default` every time for this